### ISSUE TITLE
Any way to change the Section Header/Footer Background color without Custom Views?

### ISSUE THREAD

**Issue by annjawn** [xmartlabs/Eureka#1130]
Is there a way to change the Section Header and footer background color without using any custom Views or xib?

- Environment: Eureka, Xcode 8 and iOS version 10.3.

**Comment by eliburke** [comment]
I don't know if there is a Eureka-specific/approved method, but the willDisplayHeaderView delegate does the trick for me:

```
    func tableView(_ tableView: UITableView, willDisplayHeaderView view: UIView, forSection section: Int) {
        if let header = view as? UITableViewHeaderFooterView {
            header.backgroundView?.backgroundColor = UIColor.green
            header.textLabel?.textColor = UIColor.yellow
        }
    }
```

works for headers that are shown like this: 
```
        var section = Section()
        section.header = HeaderFooterView(stringLiteral: "Ugly Header")
        form +++ section
```

**Comment by eliburke** [comment]
FWIW, there is a Eureka way using onSetupView but as of release 3.1 I think it may be broken. 
```
            var hfv = HeaderFooterView(stringLiteral: "Ugly Header")
            hfv.onSetupView = { view, _ in
                if let header = view as? UITableViewHeaderFooterView {
                     header.backgroundView?.backgroundColor = .yellow
                }
            }
            section.header = hfv
```

**Comment by EdgarDegas** [comment]
Try this:

```swift
    func tableView(
        _ tableView: UITableView,
        willDisplayHeaderView view: UIView,
        forSection section: Int
    ) {
        if #available(iOS 13.0, *) {
            view.subviews.first?.backgroundColor = .systemBackground
        } else {
            view.subviews.first?.backgroundColor = .white
        }
    }
```
